controllers/artists.js

```diff
@@ -1,23 +1,75 @@
 const Artist = require('../models/Artist');
+const HTTP_STATUS = require('../constant');
+const logger = require('../logging/logger');
 
 const createNewArtist = async (req, res) => {
-  res.json('Create new artists');
+  try {
+    const artist = await Artist.create({ ...req.body });
+    res.status(HTTP_STATUS.CREATED).json({
+      message: 'Artist created successfully',
+      data: { id: artist._id, name: artist.name }
```

Comment:
This line throws an unnecessary error. You can switch off `no-underscore-dangle` within eslint

Reply:
done on e66a553